backend/editor/entries.py

```diff
@@ -352,7 +352,11 @@ def get_all_root_nodes(self):
         Helper function used for getting all root nodes in a taxonomy
         """
         query = f"""
-            MATCH (n:{self.project_name}) WHERE NOT (n)-[:is_child_of]->() RETURN n
+            MATCH (n:{self.project_name})
+            WHERE NOT (n)-[:is_child_of]->()
+            AND NOT (n: STOPWORDS)
+            AND NOT (n: SYNONYMS)
+            RETURN n
```

Comment:
I'm not sure but what about 
```suggestion
            MATCH (n:{self.project_name}:ENTRY)
            WHERE NOT (n)-[:is_child_of]->()
            RETURN n
```
Do we need objects that are not entries ?